ament_tools/build_types/bazel.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ament_tools.build_type import BuildAction
+from ament_tools.build_type import BuildType
+
+from ament_tools.context import ContextExtender
+
+from ament_tools.helper import extract_argument_group
+
+
+class BazelBuildType(BuildType):
```

Comment:
What about the not yet implemented method `on_uninstall` defined in the base class?

Reply:
I totally forgot, thanks.  I added this now.